include/wpe/view-backend.h

```diff
@@ -128,6 +128,10 @@ WPE_EXPORT
 uint32_t
 wpe_view_backend_get_activity_state(struct wpe_view_backend*);
 
+WPE_EXPORT
+void
+wpe_view_backend_dispatch_set_scale(struct wpe_view_backend*, float);
```

Comment:
About the name -- _set_device_scale_factor() would seem more fitting.

Reply:
I guess we're setting a software scale factor based on a device scale factor, but I'm not fussed on the name and happy to change if you prefer. I just used `set_scale` as that matched the existing WebKit function.